lldb/include/lldb/Target/StopInfo.h

```diff
@@ -142,6 +142,9 @@ class StopInfo : public std::enable_shared_from_this<StopInfo> {
   static lldb::StopInfoSP
   CreateStopReasonProcessorTrace(Thread &thread, const char *description);
 
+  static lldb::StopInfoSP
```

Comment:
Do you somewhere make it obvious what a "history boundary" is? That seems a bit of a mysterious name to have no comment.

Reply:
Ah, okay, you do for eStopReasonHistoryBoundary, but that's fairly distant from this file.

Reply:
Adding a comment.